package.json

```diff
@@ -41,12 +41,12 @@
     "warning": "^2.0.0"
   },
   "peerDependencies": {
-    "history": "^1.16.0"
+    "history": "^1.17.0"
   },
   "devDependencies": {
     "babel": "^5.4.7",
     "babel-core": "^5.4.7",
-    "babel-eslint": "^3.1.23",
+    "babel-eslint": "^4.1.6",
```

Comment:
This got dropped somehow on this branch... needed to keep ESLint from randomly choking.